### ISSUE TITLE
I can't type anything in the password box to login to authenticate origin

### ISSUE THREAD

**Issue by Library4Fun** [JosefNemec/Playnite#1456]
**Describe the bug**

I'm trying to login to authenticate Origin. Under my email address there's a red wavy line. I can't click on or type anything the in the password box, so I can't login to Origin.

I updated Origin and logged to it seperately but Playnite still won't let me type anything int  he password box.

**Diagnostics ID**

Not a crash, but:
d664316e-43d9-427e-aea9-85f220bd4c31

**Comment by JosefNemec** [comment]
Can you post a screenshot please? I can't reproduce it, clicking into the text field allows me to type (it doesn't show typing caret for some reason but typing works).

![image](https://user-images.githubusercontent.com/3874087/65583860-40f33780-df80-11e9-996d-e84059ac4027.png)

**Comment by Library4Fun** [comment]
I wasn't sure how public these messages are, so I didn't want to post my email address publicly. Anyway, I've attached some screenshots with test@test.com and the same thing happened: When I LEFT click on the password box, it doesn't register my click and I can't type anything in the box (pic 1). Frantically clicking around I found out that when I RIGHT clicked on the box an "undo..." dialogue popped up (that I couldn't remove no matter where i clicked on the screen) (pic 2), but then I could left click on the password box (pic 3) 
![pic 1](https://user-images.githubusercontent.com/55770290/65607776-ecd95a80-df61-11e9-8e2e-a41d7c2bc789.jpg)
![pic 2](https://user-images.githubusercontent.com/55770290/65607778-ecd95a80-df61-11e9-8f6d-f98def46a474.jpg)
![pic 3](https://user-images.githubusercontent.com/55770290/65607779-ecd95a80-df61-11e9-997a-cd319ece9303.jpg)


and when I did it with my actual email it did finally authenticate my origin login.

**Comment by JosefNemec** [comment]
I've updated web view component we use to display login dialogs in Playnite 6. That should hopefully fix it for you, if it doesn't then I really don't know what to do with it. I'm unable to reproduce it any any of my systems.

**Comment by Library4Fun** [comment]
Thank you. I was able to work out a workaround, and it hasn't been a problem since then. I do love Playnite and use it daily. Thank you for your work.
Ken Freeman

    On Thursday, November 7, 2019, 8:35:40 AM PST, Josef Nemec <notifications@github.com> wrote:  
 
 
I've updated web view component we use to display login dialogs in Playnite 6. That should hopefully fix it for you, if it doesn't then I really don't know what to do with it. I'm unable to reproduce it any any of my systems.
